tools/deployment/make_linux_package.sh

```diff
@@ -51,7 +52,6 @@ DEBUG_PREFIX=$WORKING_DIR/debug
 
 function usage() {
   fatal "Usage: $0 -t deb|rpm -i REVISION -d DEPENDENCY_LIST
-
```

Comment:
Ok, save for the additional single line of whitespace surrounding the changes, any stray additions/deletions should now be resolved.